### ISSUE TITLE
Replays seem to be broken on latest/Hero models, spells, items broken in reply

### ISSUE THREAD

**Issue by Pratheekkn** [ValveSoftware/Dota2-Gameplay#12633]
### Description

Replays seem to be broken on latest.
this is a regression

Steps to repro:
1. Load into Dota2.
2. Download reply
3. Watch reply
4. Observe

### Example Match ID (and possibly Timestamp)

Replay

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107950959/ad8de919-8165-42af-a000-a15587ea4a73)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/107950959/80021247-934f-436c-a0ce-ef1e463052c7)

**Comment by revdepign** [comment]
Matches about 5 or 6 days ago are bugged.
Match-id: 7363815734 7363914547

**Comment by Zeron1x** [comment]
Yes, we're also having problems with replays that are around 6 days old. Would be nice to get fixed.
